sonar-project.properties

```diff
@@ -7,4 +7,4 @@ sonar.test.exclusions=test/coverage/**
 sonar.javascript.lcov.reportPaths=coverage/lcov.info
 sonar.exclusions=src/main/app.ts,src/main/development.ts,src/main/index.js,src/main/server.ts,src/main/modules/**/*,src/main/assets/**/*,src/main/routes/routes.ts,src/main/components/form/validator.ts
 sonar.organization=hmcts
-sonar.cpd.exclusions=src/main/controllers/NewJobStartDateController.ts,src/main/controllers/NoticeEndController.ts,src/main/controllers/StartDateController.ts,src/main/controllers/DobController.ts,src/main/controllers/EndDateController.ts
+sonar.cpd.exclusions=src/main/controllers/NewJobStartDateController.ts,src/main/controllers/NoticeEndController.ts,src/main/controllers/StartDateController.ts,src/main/controllers/DobController.ts,src/main/controllers/EndDateController.ts,src/main/definitions/constants.ts,src/main/controllers/CheckYourAnswersET3Controller.ts,src/main/controllers/CheckYourAnswersHearingPreferencesController.ts,src/main/controllers/CheckYourAnswersPayPensionAndBenefitsController.ts,src/main/controllers/CheckYourAnswersEarlyConciliationAndEmployeeDetailsController.ts,src/main/controllers/CheckYourAnswersContestClaimController.ts,src/main/controllers/CheckYourAnswersContactDetailsController.ts
```

Comment:
@Mike-Brain can you raise a ticket to make sure these Sonar exclusions are removed. Just so it's tracked

Reply:
Created https://tools.hmcts.net/jira/browse/RET-5418 to cover refactor of post logic and remove CYA Controllers from sonar-project.properties